bridge/engine/EngineStreamDirector.h

```diff
@@ -996,7 +996,8 @@ class EngineStreamDirector
             assert(configCost >= config.MinBitrateMargin + _slidesBitrateKbps);
             assert(configCost <= config.MaxBitrateMargin + _slidesBitrateKbps);
 
-            if (configCost >= bestConfigCost && configCost <= estimatedUplinkBandwidth)
+            const auto configIsBetter = bestConfigCost == 0 || configCost > bestConfigCost;
+            if (configIsBetter && configCost <= estimatedUplinkBandwidth)
```

Comment:
Maybe another PR but it is a bit confusing with the calculation above that OH bitrate is actual full bitrate of video. I thought first, why do we not multiple the number of streams with some video bit rate, and that is actually what is being done called overhead.